vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/config/schema/SchemaConfig.kt

```diff
@@ -20,15 +20,15 @@ import java.nio.file.Paths
 @Serializable
 data class SchemaConfig(
     /** Name of the [Schema]. */
-    val name: String = "vitrivr",
+    var name: String = "vitrivr",
```

Comment:
There is probably still a nicer way to handle this that does not require this field to be publicly writable, but that can be looked at at a later date and should not prevent us from merging this in a timely fashion.